src/Microsoft.Sbom.Api/Config/ConfigPostProcessor.cs

```diff
@@ -37,18 +41,27 @@ public void Process(IConfiguration source, IConfiguration destination, Resolutio
                 property.Attributes[typeof(System.ComponentModel.DefaultValueAttribute)]
                     is System.ComponentModel.DefaultValueAttribute defaultValueAttribute)
             {
-                SetDefautValue(destination, defaultValueAttribute.Value, property);
+                SetDefaultValue(destination, defaultValueAttribute.Value, property);
             }
 
-            // Run validators on all properties.
-            configValidators.ForEach(v => v.Validate(property.DisplayName, property.GetValue(destination), property.Attributes));
+            try
+            {
+                // Run validators on all properties.
+                configValidators.ForEach(v => v.Validate(property.DisplayName, property.GetValue(destination), property.Attributes));
+            }
+            catch (Exception ex)
+            {
+                var recorder = TelemetryRecorder.Create(destination, fileSystemUtils);
+                _ = recorder.LogException(ex);
+                throw;
```

Comment:
```suggestion
                throw ex;
```
Do we want to throw the exception ?